src/request.schema.json

```diff
@@ -18,6 +18,59 @@
   "description": "The HTTP Request",
   "additionalProperties": false,
   "properties": {
+    "url": {
+      "type": "string",
+      "description": "The path and request parameters of the request URL",
```

Comment:
Right now, the `url` value is identical with `path` (because `path` does not chop off the query string). I think it's fine to leave it the way it is because it is consistent with node's handling: https://nodejs.org/api/http.html#http_message_url